IdentityCore/src/cache/metadata/MSIDAccountMetadataCacheAccessor.h

```diff
@@ -34,6 +34,8 @@
 
 @interface MSIDAccountMetadataCacheAccessor : NSObject
 
+@property (nonatomic) BOOL skipCacheForAccountMetadata;
```

Comment:
will it be less ambiguous if we name it `skipMemoryCacheForAccountMetadata`? I think `cache` could mean keychain as well :P

Reply:
+1

Reply:
I like it, changed.